exercises/atbash-cipher/src/test/java/AtbashTest.java

```diff
@@ -85,5 +85,18 @@ public void testDecodeAllTheLetters() {
         assertEquals("thequickbrownfoxjumpsoverthelazydog",
                 atbash.decode("gsvjf rxpyi ldmul cqfnk hlevi gsvoz abwlt"));
     }
-
+    
+    @Ignore("Remove to run test")
+    @Test
+    public void testDecodeWithTooManySpaces() {
+        assertEquals("exercism",
+                atbash.decode("vc vix    r hn"));
```

Comment:
Not very important but this could probably all fit on one line? We usually only split things over multiple lines if it's too long for one line. Exactly what people think "too long" is can vary, but in general 120 characters can be a good cut off point 🙂

Reply:
done